src/redux/states.js

```diff
@@ -0,0 +1,22 @@
+export let todos =[
```

Comment:
There is no `complete` state for todos. We removed the complete functionality from the app